BUILD

```diff
@@ -178,6 +178,9 @@ DEFAULT_STOP_TIMEOUT=XXX
     this, its process group is sent a SIGKILL signal which should cause it to terminate immediately.
     The default if unspecified is 10 seconds. (The value can be overridden for individual services
     via the service description).
+SUPPORT_SELINUX=1|0
+    Whether to build support for SELinux awareness in dinit, such as loading the system SELinux policy
+    at boot.
```

Comment:
Since all it does right now is load the policy, I think that's what this should say. (I understand that you want to implement more later and that's fine, the documentation here can be updated accordingly at that time; for now, I think it should be precise).

Reply:
That makes sense, will do

Reply:
Done